tiledb/sm/serialization/array_schema.h

```diff
@@ -52,22 +52,28 @@ class Buffer;
 class ArraySchema;
 class Dimension;
 class MemoryTracker;
+class URI;
 enum class SerializationType : uint8_t;
 
 namespace serialization {
 
 class LoadArraySchemaRequest {
  public:
-  LoadArraySchemaRequest(bool include_enumerations = false)
-      : include_enumerations_(include_enumerations) {
+  explicit LoadArraySchemaRequest(const Config& config)
+      : config_(config) {
+  }
+
+  inline const Config& config() const {
+    return config_;
   }
 
   inline bool include_enumerations() const {
-    return include_enumerations_;
+    return config_.get<bool>(
+        "rest.load_enumerations_on_array_open", config_.must_find);
```

Comment:
How about we read this value on construction? There is no reason to copy a whole config.

Reply:
Done